src/extension.ts

```diff
@@ -37,9 +40,20 @@ export function activate(context: vscode.ExtensionContext) {
 
     onboardingWizard.registerCommand(context);
     onboardingWizard.onActivate(context);
+    settingsCommand.registerCommand(context);
 
     configureLintingToolsCommand.registerCommand(context);
-    lspClient.activate(context);
+
+    const extensionId = getExtensionId(context);
+    const command = getUpdateDiagnosticsSettingCommand(context);
+    const section = getDiagnosticsSettingSection(context);
+
+    lspClient.activate(
```

Comment:
pass in three 3 strings into client, then server to avoid hardcoding in multiple place.